openbas-api/src/test/java/io/openbas/utils/mockUser/WithMockAdminUser.java

```diff
@@ -0,0 +1,14 @@
+package io.openbas.utils.mockUser;
+
+import org.springframework.security.test.context.support.WithSecurityContext;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import static io.openbas.utils.mockUser.WithMockAdminUserSecurityContextFactory.MOCK_USER_ADMIN_EMAIL;
+
+@Retention(RetentionPolicy.RUNTIME)
+@WithSecurityContext(factory = WithMockAdminUserSecurityContextFactory.class)
+public @interface WithMockAdminUser {
```

Comment:
@WithMockUser(roles = {"ADMIN"}) is not enough ?

Reply:
I tried but I had this error: java.lang.ClassCastException: class org.springframework.security.core.userdetails.User cannot be cast to ( are in unnamed module of loader 'app')